keyboards/boardsource/5x12/config.h

```diff
@@ -4,7 +4,7 @@
 
 /* USB Device descriptor parameter */
 #define VENDOR_ID       0xF7E0
-#define PRODUCT_ID      0x0412
+#define PRODUCT_ID      0x5012
```

Comment:
General comment:

The `3x4` uses `0x0304` here, and the `4x12` uses `0x0412` - why `0x5012` (50x12?) here?

Reply:
0x0512 collides with keycapsss/o4l_5x12

since that's with a different vendor, would it be okay to use it here?

Reply:
The `keycapsss/o4l_5x12` uses `0x7983` as the Vendor ID. For VIA's purposes, it's the pairing of the Vendor and Product IDs that matter - two devices can use the same Product ID if the Vendor IDs are different, and can use the same Vendor IDs if the Product IDs are different.

Reply:
okay! sounds good. Updated in commit https://github.com/qmk/qmk_firmware/pull/10236/commits/7b5e44300891ca87958da3b068222b933d88aba2